### ISSUE TITLE
settings are still resetting

### ISSUE THREAD

**Issue by jerkstore369** [iina/iina#2272]
Once in a while I'll check the quick panel settings and the video settings I've set will be at defualt. WTF, how hard is it to ensure that the bloody settings are saved? This has been ongoing since god knows how long. I like to have custom gamma and contrast as well as audio equalizer settings. Who knows what triggers the reset but it's ridiculous. Get this crap working, man. Jesus.

**Comment by saagarjha** [comment]
We're generally happy to help, but we'd appreciate it if you didn't berate us :(

**Comment by jerkstore369** [comment]
I'm not berating, I'm just really annoyed. This has been an issue for god knows how long.

**Comment by saagarjha** [comment]
Yeah, I get that, but it would be much more productive you added more details about your issue rather than being mean…

**Comment by jerkstore369** [comment]
I don't know what other details to add. I just check quick panel settings once in a while and every time I do the settings are reset to default. Should I check every time I run IINA, and then when it changes, log out, log in as admin, then check the console. Honestly, I am so close to buying a PC laptop and installing LINUX.

**Comment by saagarjha** [comment]
Are you seeing this happen in between videos, while watching the same video, after not using IINA for a while, etc…

**Comment by jerkstore369** [comment]
I make the changes. I watch a number of videos over the span of a few weeks or whatever, then when I check the equalizer, the settings are reset to default.

**Comment by lhc70000** [comment]
@jerkstore369 All settings in the sidebar only work for the **current video** (for example you wouldn’t like all videos having the same delay for subtitles, right?), and only those in the preferences panel are global. Any options you set in the sidebar will be saved together with the playback progress, but they won’t persist if you already finished watching the video.

If you really need global settings for contrast etc, currently you can add extra mpv options in Preferences - Advanced. To be specific, add an option with key=“contrast” and value=“60” or any value you want, then restart IINA.

**Comment by jerkstore369** [comment]
It would be helpful if there was a checkbox or something to make certain setting global.

**Comment by MHX792** [comment]
Computer screens are usually dark. I have always been increasing the gamma value in video players. Having to do this over and over for every new video is tiring. We need global settings for this otherwise awesome app.

**Comment by dxmind** [comment]
Is it possible to add extra mpv options for an audio EQ? Or to apply saved audio filters automatically? Btw, I've noticed that if I apply an audio filter, it affects the sound, but visually the equalizer doesn't change.
